WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -856,6 +856,14 @@ extension UIImage {
         return UIImage(imageLiteralResourceName: "megaphone").imageFlippedForRightToLeftLayoutDirection()
     }
 
+    /// Speaker icon
+    ///
+    static var speakerIcon: UIImage {
+        return UIImage.gridicon(.speaker)
+            .withRenderingMode(.alwaysTemplate)
+            .imageFlippedForRightToLeftLayoutDirection()
```

Comment:
nit: it could be better to use this method directly where the image is used. In this way, the image is more reusable in the feature and we don't apply any significant change to the original image.

Reply:
Good point! Updated in 23e11d6d7842912aba831c3d95e8500b169c740c.